hs/CourseographyFacebook.hs

```diff
@@ -85,23 +97,24 @@ performFBAction action = withManager $ \manager -> FB.runFacebookT credentials m
 -- | Posts a message to the user's Facebook feed, with the code 'code'. GraphResponse
 -- is then sent back to the user.
 postToFacebook :: String -> ServerPart Response
-postToFacebook code = (liftIO $ performPost code) >> graphResponse
+postToFacebook code = (liftIO $ performPost (BS.pack code)) >> graphResponse
```

Comment:
How so? `code` needs to be a bytestring. I don't believe that there is a better way to do this considering code as a Text, as it will eventually need to be converted to BS.

There is a function, `lookBS`, but I have not been able to get it to work as it takes in a different ByteString (lazy?).